packages/typescript/CHANGELOG.md

```diff
@@ -6,22 +6,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.0]
+### Added
+- Add more setup instructions to `@metamask/typescript-eslint-config` README ([#209](https://github.com/MetaMask/eslint-config/pull/209))
+- **BREAKING:** Add rules that require type information ([#250](https://github.com/MetaMask/eslint-config/pull/250))
```

Comment:
Nit: Usually we group all of the breaking changes together at the beginning of the section, since those are usually what people are most interested in reading